benchmark/_benchmark_progress.js

```diff
@@ -40,7 +40,7 @@ class BenchmarkProgress {
     this.completedConfig = 0;
     // Total number of configurations for the current file
     this.scheduledConfig = 0;
-    this.interval = 0;  // Result of setInterval for updating the elapsed time
+    this.interval;  // Updates the elapsed time.
```

Comment:
Did you mean to remove the assignment?

Reply:
Yes, the default value was misleading. It will only contain an interval so it can clear that interval later on again. Shall I move that into a separate commit?

Reply:
Asking because the statement is now a no-op and could be deleted

Reply:
That is completely true. The hidden class would change if removed (that might be the case anyway due to the type change).

I personally would keep it around for the comment and clarity about instance variables.

Reply:
Just accessing the property changes the hidden class? I'm not sure about that. What I would prefer is the value tu be initialized to something. It can be null if you want

Reply:
Oh, my bad. I am still sleepy. Hm, there is another access similar to this one: https://github.com/nodejs/node/pull/24996/files#diff-d270293890ef0a5137a4c5c0e31afdfcL38

I guess the only reason to keep them is to "visualize" that these are instance variables set later on.

Reply:
OK 🤷‍♂️